src/abstractdataframe/abstractdataframe.jl

```diff
@@ -448,68 +456,58 @@ function StatsBase.describe(io::IO, df::AbstractDataFrame; stats::AbstractVector
     return data
 end
 
-# Define 4 functions for getting summary statistics 
+# Define functions for getting summary statistics 
 # use a dict because we dont know which measures the user wants
 # Outside of the `describe` function due to something with 0.7
-function get_stats(col::AbstractArray{<:Real})
-    sumstats = summarystats(col)
-    Dict(
-        :mean => sumstats.mean,
-        :std => Compat.std(col, mean = sumstats.mean),
-        :min => sumstats.min,
-        :q25 => sumstats.q25,
-        :median => sumstats.median,
-        :q75 => sumstats.q75,
-        :max => sumstats.max,
-        :nmissing => nothing,
-        :nunique => nothing,
-        :eltype => eltype(col)
-    )
-end
 
-function get_stats(col::AbstractArray{<:Union{Real, Missing}})
+function get_stats(col::AbstractArray{>:Missing})
     nomissing = collect(skipmissing(col))
-    sumstats = summarystats(nomissing)
-    Dict(
-        :mean => sumstats.mean,
-        :std => Compat.std(nomissing, mean = sumstats.mean),
-        :min => sumstats.min,
-        :q25 => sumstats.q25,
-        :median => sumstats.median,
-        :q75 => sumstats.q75,
-        :max => sumstats.max,
-        :nmissing => count(ismissing, col),
-        :nunique => nothing,
-        :eltype => Missings.T(eltype(col))
-    )
-end
+    
+    q = try quantile(nomissing, [.25, .5, .75]) catch [nothing, nothing, nothing] end
+    ex = try extrema(nomissing) catch (nothing, nothing) end
+    if typeof(nomissing) <: AbstractArray{<:Real}
```

Comment:
`eltype(nomissing) <: Real` would be simpler, and the `?` ternary syntax should be fine here.